### ISSUE TITLE
Ensure exceptions are making it to rollbar.com in production

### ISSUE THREAD

**Issue by pbiggar** [darklang/dark#3434]
It seems that exceptions are not making it to rollbar.com. The exceptions are being recorded by telemetry (honeycomb) however.

I triggered some exceptions a few days ago from ExecHost and they made it to rollbar.

Exceptions from editor-deployment are failing with: "invalid access token". They were using the old access token that I recently updated. However, those haven't actually worked for literally years, and it's not clear what causes it. I'm redeploying and I'll see what outcome I get.

The firewall is open to access https://api.rollbar.com on exechost, and I managed to trigger a rollbar from the command line by making a json request with curl.

- [x] Validate rollbar *notifications* can reach from ExecHost
- [ ] Validate rollbars can reach from
  - [x] ApiServer
  - [ ] bwdserver
  - [x] exechost
  - [ ] queueworker
  - [x] cronchecker

**Comment by pbiggar** [comment]
After some more debugging, i can consistently get rollbars to work using the blocking API on ExecHost. I also managed to get them to work with the non-blocking API if I kept the application from terminating, which indicates that they just take a bit of time to propagate.

I've emailed rollbar support asking how to ensure all exceptions are propagated before we shut down the process.

For the asp.net servers, they have a shutdown timer of 28seconds, so we're unlikely to have a problem with them.

For the console servers (cronchecker and queueworker), we should definitely wait.
